### ISSUE TITLE
wrong xadesSignedPropertiesDigestValue

### ISSUE THREAD

**Issue by akbaras** [marwan2/zatcaIntegration#4]
I am getting below error when validating xml file. Can anyone suggest what I am missing?

![image](https://user-images.githubusercontent.com/53428958/208436470-8305dd5e-055e-4dad-9ada-4ed9d01da5aa.png)

category : SIGNATURE_ERROR
code :xadesSignedPropertiesDigestValue
message : wrong xadesSignedPropertiesDigestValue

**Comment by marwan2** [comment]
Ensure you generated properly CSR
then used this valid CSR to generate Compliance CSID & Production CSID
and for sure save them for future usage

**Comment by akbaras** [comment]
Could you please share config (.cnf) file to generate CSR?

I have follow your steps but get expired certificate from Production CSID response

**Comment by akbaras** [comment]
@marwan2   I have follow your steps and got certificate, but it's getting Invalid to me. 

![image](https://user-images.githubusercontent.com/53428958/208897030-6fdcfde2-f074-4c67-886f-4ccf251cd16a.png)

**Comment by marwan2** [comment]
@akbaras, I found same content in my Production CSID, may be due to using Sandbox environment. 
But I can still use it for compliance/reporting invoices in Sandbox APIs.

**Comment by akbaras** [comment]
@marwan2   So do you get any error while validating xml?
Like:

![image](https://user-images.githubusercontent.com/53428958/208916342-38e09bcd-e557-449f-a5e5-10de9ea5246e.png)

**Comment by marwan2** [comment]
@akbaras No, recently I got validation errors due to some calculations, but not the signature
